vue-frontend/src/components/contributions/GithubContribution.vue

```diff
@@ -123,25 +123,27 @@
     <!-- Mobile UI end -->
     <!-- Desktop UI start -->
     <div
-      class="tw-container tw-hidden lg:tw-flex tw-flex-wrap tw-justify-between tw-mx-auto tw-my-10"
+      class="lg:tw-container tw-hidden md:tw-flex tw-flex-wrap tw-justify-between tw-mx-auto lg:tw-my-10"
     >
       <div
         v-for="(contribution, index) in contributions"
         class="flip-card tw-relative tw-w-[48%] tw-h-[440px] xl:tw-h-[500px] 2xl:tw-h-[550px] tw-mt-[2rem] tw-mx-auto tw-cursor-pointer"
         :key="index"
         @click="openContribution(contribution)"
       >
-        <div class="flip-card-inner tw-relative tw-w-full tw-h-full">
+        <div
+          class="flip-card-inner tw-relative tw-w-full tw-h-[92%] lg:tw-h-full"
+        >
           <div class="flip-card-front">
             <img
               :src="contribution.deskImage[1]"
               :srcset="`${contribution.deskImage[0]} 400w, ${contribution.deskImage[1]} 800w`"
-              class="tw-absolute tw-w-full tw-h-full tw-object-cover tw-rounded-[20px] card-image card-image-1"
+              class="tw-absolute tw-w-full lg:tw-h-full tw-object-cover tw-object-contain tw-rounded-[20px] card-image card-image-1"
               alt="contribution-image"
             />
           </div>
           <div
-            class="flip-card-back tw-absolute tw-w-full tw-h-full tw-rounded-[20px] tw-from-[#FF835B] tw-to-[#F2709C] tw-bg-gradient-[180deg] card-image back-card"
+            class="flip-card-back tw-absolute tw-w-full tw-h-[92%] lg:tw-h-full tw-rounded-[20px] tw-from-[#FF835B] tw-to-[#F2709C] tw-bg-gradient-[180deg] card-image back-card"
```

Comment:
Fix following UI break

![Screenshot from 2023-06-21 17-30-30](https://github.com/canopas/canopas-website/assets/69897605/0b96f2c7-77fb-413e-9e60-30ebbfc3ebba)

Reduce verticle space between cards in md devices
![Screenshot from 2023-06-21 17-31-12](https://github.com/canopas/canopas-website/assets/69897605/d1ebcf67-1f9d-4c9e-a682-0bc12ae2cc79)